pkg/apis/kops/componentconfig.go

```diff
@@ -314,6 +314,8 @@ type KubeAPIServerConfig struct {
 	ServiceClusterIPRange string `json:"serviceClusterIPRange,omitempty" flag:"service-cluster-ip-range"`
 	// Passed as --service-node-port-range to kube-apiserver. Expects 'startPort-endPort' format e.g. 30000-33000
 	ServiceNodePortRange string `json:"serviceNodePortRange,omitempty" flag:"service-node-port-range"`
+	// EgressSelectorConfigFile is the file path with api-server egress selector configuration.
+	EgressSelectorConfigFile string `json:"egressSelectorConfigFile,omitempty" flag:"egress-selector-config-file"`
```

Comment:
```suggestion
	EgressSelectorConfigFile string `json:"-" flag:"egress-selector-config-file"`
```

I see in the baremetal PR that there's a plan for kOps to manage this file. So is there a need to have the admin configure it?